### PR TITLE
bootstrap@4.0.0-alpha

### PR DESCRIPTION
**Pull request by jdanyow**

### CODE REVIEW

File: package-overrides/github/twbs/bootstrap@4.0.0-alpha.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "main": "js/bootstrap",
+  "files": [
+    "dist/js",
+    "dist/css"
+  ],
+  "ignore": [
+    "dist/js/umd",
```
Comment (guybedford) [inline package-overrides/github/twbs/bootstrap@4.0.0-alpha.json]: Perhaps we should keep these available for modular use?

File: package-overrides/github/twbs/bootstrap@4.0.0-alpha.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "main": "js/bootstrap",
+  "files": [
+    "dist/js",
+    "dist/css"
+  ],
```
Comment (guybedford) [inline package-overrides/github/twbs/bootstrap@4.0.0-alpha.json]: I'm wondering if we don't just remove this entirely and set `files: null`.

**Comment by guybedford** [comment]
Thanks so much for putting this together! Note that this override is having to "cancel out" the override already present in https://github.com/twbs/bootstrap/blob/master/package.js. If you're able to provide a corresponding PR to bootstrap that would be a great help, otherwise I'll go ahead and do this when we merge.

File: package-overrides/github/twbs/bootstrap@4.0.0-alpha.json (hunk 1 above)
Comment (jdanyow) [inline package-overrides/github/twbs/bootstrap@4.0.0-alpha.json]: sounds good

File: package-overrides/github/twbs/bootstrap@4.0.0-alpha.json (hunk 2 above)
Comment (jdanyow) [inline package-overrides/github/twbs/bootstrap@4.0.0-alpha.json]: ok will give that a try

**Comment by jdanyow** [comment]
I can create a PR for the bootstrap repo- I should be modifying this file right:
https://github.com/twbs/bootstrap/blob/v4-dev/package.json#L83

(you linked to their `package.js` in master above- I think that was a typo but just wanted to make sure).

**Comment by guybedford** [comment]
Thanks, yes exactly, it should be on the 4 dev branch. Just let me know if you have any further questions at all here.

**Comment by jdanyow** [comment]
I've made the changes you mentioned and created a PR in the bootstrap v4 alpha branch.  I removed the `"files": null` in the bootstrap repo PR.

Let me know if any other tweaks are required.

Thanks!

**Comment by guybedford** [comment]
Great thanks for the update.

**Pull request merged**